Dockerfile.benchmark

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.69.0@sha256:c2dcc19371a976065950915307b03a9c5514e5143bdd833635d5501a0642992e
+FROM rust:1.77.2@sha256:da23629cc6826f0c395c8697994cdce0b5ac6850a695c448a101c3cc9cd6a59b
```

Comment:
Fixes:

```
#8 69.41 error: failed to compile `hyperfine v1.18.0`, intermediate artifacts can be found at `/tmp/cargo-installVcS5Gb`
#8 69.41 
#8 69.41 Caused by:
#8 69.41   package `anstyle-parse v0.2.3` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.69.0
#8 69.41   Try re-running cargo install with `--locked`
#8 ERROR: process "/bin/sh -c cargo install hyperfine" did not complete successfully: exit code: 101
```